extensionsGallery-insider.json

```diff
@@ -5206,6 +5206,66 @@
 					],
 					"statistics": [],
 					"flags": ""
+			},{
+					"extensionId": "102",
+					"extensionName": "storeprocdure2SQL",
+					"displayName": "Store Procdure To SQL",
+					"shortDescription": "tore Procdure,Convert",
+					"publisher": {
+						"displayName": "lozn",
+						"publisherId": "lozn",
+						"publisherName": "lozn"
+					},
+					"versions": [
+						{
+							"version": "1.0.0",
+							"lastUpdated": "08/23/2023",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/lozn00/AzureSQLProcConvertSQL/raw/master/StoredProcedureConverter-0.0.1.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://raw.githubusercontent.com/lozn00/AzureSQLProcConvertSQL/master/png/search.png"
```

Comment:
This icon image does not exist. Please either change this to point to an icon that exists or add it to your repo.